src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,45 @@
+package ui;
+
+/**
+ * Deaks with interactions with the user
+ */
+public class Ui {
+    /**
+     * Constructor is not needed
+     */
+    private Ui() {
+        throw new AssertionError("Constructor is not allowed");
+    }
+
+    /**
+     * Print out welcome speech
+     */
+    public static void welcomeText() {
+        System.out.println("___________________________________________");
+        System.out.println("Hello! I'm Taylor");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Print out the error message
+     * @param err Error thrown from ChatBot
+     */
+    public static String printError(Exception err) {
+        return ("Error: " + err);
+    }
+
+    /**
+     * Print warning for invalid commands from Users
+     */
+    public static String invalidCommand() {
+        return ("Invalid input. ChatBot can only handle "
```

Comment:
Like that you told the user the commands that are available! Perhaps you could also provide the commands' formats (e.g. mark <task_number>)?